adds tasks/status.sh

mcka1n opened the pull request:
### Changes

I'm adding a Task called `status.sh` to provide the number of Bolt processes that are running, and details about memory.

Example:

**Run the Task/script**

```
cd /etc/puppetlabs/code/environments/production/modules/puppet_bolt_server/tasks
bash status.sh
```


**Output**
```
{
    "number_bolt_processes": "198",
    "free_memory": [
        {
            "type": "RAM",
            "total": 15833,
            "used": 6999,
            "free": 7587,
            "shared": 24,
            "buffers": 2,
            "cache": 1243,
            "available": 8505
        },
        {
            "type": "Swap",
            "total": 0,
            "used": 0,
            "free": 0
        }
    ]
}
```

mcka1n committed:
adds tasks/bolt_server_status.sh

mcka1n committed:
removes quotes to fix JSON

timidri reviewed (changes requested):
I suggest renaming the task to `status` so the full name will be 
`puppet_bolt_server::status` instead of `puppet_bolt_server::bolt_server_status`

mcka1n committed:
renames bolt_server_status task to just status.sh

mcka1n committed:
uses free -h flag to show human-readable output

mcka1n commented:
@timidri I changed the Task name based on your suggestion 👍 

**Bonus**: The `free` output also changed, I'm using the human-readable format (in GBs).

```
{
    "number_bolt_processes": "199",
    "free_memory": [
        {
            "type": "RAM",
            "total": 15Gi,
            "used": 7.0Gi,
            "free": 7.3Gi,
            "shared": 24Mi,
            "buffers": 2.0Mi,
            "cache": 1.2Gi,
            "available": 8.2Gi
        },
        {
            "type": "Swap",
            "total": 0B,
            "used": 0B,
            "free": 0B
        }
    ]
}
```
